### PR TITLE
add roadmap

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
Signed-off-by: xiang <xiang13225080@163.com>

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Chaosd Roadmap
+
+This document defines the roadmap for Chaosd development.
+
+## v2.0
+
+- [ ] Support HTTP attack to simulate HTTP faults, such as abort connection, delay, etc.
+- [ ] Support IO attack to simulate file system faults, such as IO delay and read/write errors.
+- [ ] Support workflow to manage a group of chaos experiments.
+- [ ] Support use Dashboard to manage chaos experiments.
```
Comment (Andrewmatilde) [inline ROADMAP.md:10]: I think dashboard may be hard to be supported in v2.0.

Comment (WangXiangUSTC) [reply ROADMAP.md]: done

**Comment by WangXiangUSTC** [comment]
@Andrewmatilde @cwen0 PTAL again

**Review by Andrewmatilde — APPROVED** [review]
LGTM

**Comment by Andrewmatilde** [comment]
PTAL, @cwen0

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**